extensions/sql-bindings/src/sqlbindings.d.ts

```diff
@@ -0,0 +1,97 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+
+declare module 'sqlbindings' {
+
+	/**
+	 * sql bindings extension
+	 */
+	export interface IExtension {
+		/**
+		 * Adds a SQL Binding to a specified Azure function in a file
+		 * @param bindingType Type of SQL Binding
+		 * @param filePath Path of the file where the Azure Functions are
+		 * @param functionName Name of the function where the SQL Binding is to be added
+		 * @param objectName Name of Object for the SQL Query
+		 * @param connectionStringSetting Setting for the connection string
+		 */
+		addSqlBinding(bindingType: BindingType, filePath: string, functionName: string, objectName: string, connectionStringSetting: string): Promise<ResultStatus>;
+
+		/**
+		 * Gets the names of the Azure functions in the file
+		 * @param filePath Path of the file to get the Azure functions
+		 * @returns array of names of Azure functions in the file
+		 */
+		 getAzureFunctions(filePath: string): Thenable<GetAzureFunctionsResult>;
```

Comment:
This should be returning a promise. For things that call Thenable APIs you can just make the function implementation async and await the Thenable call and return the value - which will then wrap it in a Promise.

Reply:
Gotcha, I did that for the implementation but forgot to change this here when moving over.